TTS/vocoder/configs/multiband-melgan_and_rwd_config.json

```diff
@@ -138,6 +138,8 @@
     "print_eval": false,     // If True, it prints loss values for each step in eval run.
     "save_step": 25000,      // Number of training steps expected to plot training stats on TB and save model checkpoints.
     "checkpoint": true,     // If true, it saves checkpoints per "save_step"
+    "keep_best": false,  // If true, keeps all best_models after keep_after steps
```

Comment:
Right now what happens if keep_best: false? Does it keep any best_model at all in that case?

Reply:
It will keep just the last best model deleting previous ones. It's a bit misleading you're right - maybe we change it to `keep_all_best`?

Reply:
I think ```keep_all_best``` is more intuitive.